lib/index.ts

```diff
@@ -297,18 +299,22 @@ export async function initializeBuildMetadata(
 	secrets: SecretsPopulationMap;
 	regSecrets: RegistrySecrets;
 	architecture: string;
+	apiVersion: semver.SemVer;
```

Comment:
make this string? and coerce here: [`e869323` (#80)](https://github.com/balena-io-modules/resin-multibuild/pull/80/commits/e8693233c2b26202619c4830fa8391cbf3a2900e#diff-13b5b151431c7e7a17f273559ed212d5R272)

Reply:
`semver.SemVer | null`